directory.py

```diff
@@ -0,0 +1,69 @@
+from Person import Person
+from json_reader import JSONReader
+
+class Directory:
```

Comment:
All the Primary Actions and the Special Features should be provided by the Directory Class. Seems like the Primary Actions are not implemented here